apps/judicial-system/api/src/app/modules/case-list/interceptors/caseList.interceptor.ts

```diff
@@ -48,34 +27,10 @@ export class CaseListInterceptor implements NestInterceptor {
               isValidToDateInThePast: theCase.validToDate
                 ? Date.now() > new Date(theCase.validToDate).getTime()
                 : theCase.isValidToDateInThePast,
-              appealedDate: getAppealedDate(
-                theCase.prosecutorPostponedAppealDate,
-                theCase.accusedPostponedAppealDate,
-              ),
-              // This state overwrite is added in at least temporarily to handle strange
-              // behaviour when a case is reopened after being appealed and then closed
-              // again with everyone having accepted the ruling in court
-              appealState: !wasAcceptedInCourt(
-                theCase.prosecutorAppealDecision,
-                theCase.accusedAppealDecision,
-              )
-                ? theCase.appealState
-                : undefined,
             }
           }
 
-          const { indictmentRulingDecision, rulingDate, defendants } = theCase
-
-          const indictmentInfo = getIndictmentInfo({
-            indictmentRulingDecision,
-            rulingDate,
-            defendants,
-          })
-
-          return {
-            ...theCase,
-            ...indictmentInfo,
-          }
+          return theCase
```

Comment:
This intercepter is deprecated, is only used for defender tables and will be removed when defender tables have been move to the new table architecture. This PR removes some code that is not needed for the current defender table.